### ISSUE TITLE
[Bug]: prowler AWS --list-compliance-requirements results in error/ no requirements for certain frameworks

### ISSUE THREAD

**Issue by edschreibman** [toniblyx/prowler#2071]
**What happened?**
prowler aws --list-compliance-requirements  results in error/ no requirements the frameworks listed below

**How to reproduce it**
1. run `prowler aws --list-compliance` to get list of frameworks
2. run `prowler aws --list-compliance-requirements [framework]`
3. For these Frameworks:
i.e. `prowler aws --list-compliance-requirements soc2_aws`
- soc2_aws
- cisa_aws
- ffiec_aws
- gdpr_aws
- hipaa_aws

error produced:

```
		Traceback (most recent call last):
  File "/usr/local/bin/prowler", line 8, in <module>
    sys.exit(prowler())
  File "/usr/local/lib/python3.9/site-packages/prowler/__main__.py", line 98, in prowler
    print_compliance_requirements(
  File "/usr/local/lib/python3.9/site-packages/prowler/lib/check/check.py", line 187, in print_compliance_requirements
    provider = split_compliance[2].upper()
IndexError: list index out of range
```
For these frameworks:

- aws_audit_manager_control_tower_guardrails_aws
- aws_foundational_security_best_practices_aws
- fedramp_low_revision_4_aws
- fedramp_moderate_revision_4_aws
- gxp_eu_annex_11_aws
- gxp_21_cfr_part_11_aws
- nist_800_53_revision_4_aws
- nist_800_53_revision_5_aws
- nist_800_171_revision_2_aws
- nist_csf_1.1_aws
- rbi_cyber_security_framework_aws

			
No error - But no requirements in output
 

**Expected behavior**

Output to contain controls in framework, like this for **ens_rd2022_aws**:
```
Listing ENS RD2022 AWS Compliance Requirements:

Requirement Id: [35mop.acc.1.aws.iam.2[0m
	- Description: Proveedor de identidad centralizado
	- Checks:
 [33m		iam_check_saml_providers_sts
[0m
Requirement Id: [35mop.acc.2.aws.iam.4[0m
	- Description: Requisitos de acceso
	- Checks:
 [33m		iam_avoid_root_usage
[0m
Requirement Id: [35mop.acc.3.r1.aws.iam.1[0m
	- Description: Segregación rigurosa
	- Checks:
 [33m		iam_support_role_created
```

**Screenshots or Logs**
<img width="413" alt="1" src="https://user-images.githubusercontent.com/102334064/224187021-72eb6e71-35f3-414e-82c8-0305f61c0b54.png">
<img width="765" alt="2" src="https://user-images.githubusercontent.com/102334064/224187029-2e256bd6-6de1-4040-bcab-2c2e5da35a9b.png">
<img width="786" alt="3" src="https://user-images.githubusercontent.com/102334064/224187049-7efd4bc5-d4fa-408e-850c-0159d3167ada.png">
<img width="637" alt="4" src="https://user-images.githubusercontent.com/102334064/224187064-ed9e0ba5-4b75-4572-8465-5378637b3a77.png">

Also, you can add logs (anonymize them first!). Here a command that may help to share a log
`prowler <your arguments> --log-level DEBUG --log-file $(date +%F)_debug.log` then attach here the log file.

**From where are you running Prowler?**
Please, complete the following information:
 - Resource: (e.g. EC2 instance,  )  : EC2 Instance
 - OS: [e.g. Amazon Linux 2  ]: Amazon Linux 2  ]
prowler --version Prowler 3.2.0
python --version  Python 2.7.18
pip --version pip 23.0 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)

 - Installation method (Are you running it from pip package or cloning the github repo?):  #Install Prowler via pip3             /usr/local/bin/pip3 install prowler
 
 - Others:

**Additional context**

**Comment by MrCloudSec** [comment]
Hi @edschreibman, can you use the last version of prowler `pip install --upgrade prowler`? These errors are not present in version 3.2.4.

**Comment by edschreibman** [comment]
Tested and agreed - fully resolved in Prowler 3.2.4
Fri Mar 10 23:45:28 UTC 2023
Prowler 3.2.4
